src/main/resources/admin/tools/content-type-manager/components/component-file-handler.html

```diff
@@ -26,41 +26,62 @@
       },
       methods: {
         async createNewContentType() {
-          alert("Please select the content-types folder.");
+          Swal.fire({
+            text: "Please select the content-types folder.",
+            showCancelButton: true,
+          }).then(async (result) => {
+            if (!result.isConfirmed) return;
 
-          dirHandle = await window.showDirectoryPicker();
-          const contentTypeName = window.prompt(
-            "What is the name of the content type?"
-          );
-          const newContentTypeDirectoryHandle =
-            await dirHandle.getDirectoryHandle(contentTypeName, {
-              create: true,
+            dirHandle = await window.showDirectoryPicker();
+
+            const { value: contentTypeName } = await Swal.fire({
+              text: "What is the name of the content type?",
+              input: "text",
             });
-          this.fileHandle = await newContentTypeDirectoryHandle.getFileHandle(
-            `${contentTypeName}.xml`,
-            { create: true }
-          );
-        },
-        async getFileContent() {
-          alert("Please select the xml file of a content-type.");
 
-          [fileHandle] = await window.showOpenFilePicker();
+            if (!contentTypeName) return;
 
-          const file = await fileHandle.getFile();
+            const newContentTypeDirectoryHandle =
+              await dirHandle.getDirectoryHandle(contentTypeName, {
+                create: true,
+              });
 
-          if (file.type !== "text/xml") {
-            alert("This is not a XML file.");
-            return false;
-          }
+            this.fileHandle = await newContentTypeDirectoryHandle.getFileHandle(
+              `${contentTypeName}.xml`,
+              { create: true }
+            );
+          });
+        },
+        async getFileContent() {
+          Swal.fire({
+            text: "Please select the xml file of a content-type.",
+            showCancelButton: true,
+          }).then(async (result) => {
+            if (!result.isConfirmed) return;
+
+            [fileHandle] = await window.showOpenFilePicker();
+
+            const file = await fileHandle.getFile();
+
+            if (file.type !== "text/xml") {
+              Swal.fire({
+                text: "This is not a XML file.",
+                icon: "error",
+                showConfirmButton: false,
+                timer: 1500,
+              });
+              return false;
+            }
 
-          this.setJson(await file.text());
+            this.setJson(await file.text());
 
-          this.fileHandle = fileHandle;
+            this.fileHandle = fileHandle;
 
-          Swal.fire({
-            icon: "success",
-            showConfirmButton: false,
-            timer: 1500,
+            Swal.fire({
+              icon: "success",
+              showConfirmButton: false,
+              timer: 1500,
+            });
```

Comment:
Seems you are doing a bunch of useful things. 
Choosing a folder to get the files from. 
Then selecting the file and checking if its an xml 👍 
But this is not realted to the issue #3?